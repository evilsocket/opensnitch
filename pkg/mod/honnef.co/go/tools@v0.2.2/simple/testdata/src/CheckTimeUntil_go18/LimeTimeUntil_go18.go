package pkg

import "time"

func fn(t time.Time) {
	t.Sub(time.Now()) // want `time\.Until`
	t.Sub(t)
	t2 := time.Now()
	t.Sub(t2)
}
