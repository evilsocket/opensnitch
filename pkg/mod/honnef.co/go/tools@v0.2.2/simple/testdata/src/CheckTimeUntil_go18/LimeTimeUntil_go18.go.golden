package pkg

import "time"

func fn(t time.Time) {
	time.Until(t) // want `time\.Until`
	t.Sub(t)
	t2 := time.Now()
	t.Sub(t2)
}
